app/src/main/java/com/scouts/kitchenplaner/datalayer/repositories/ShoppingListRepository.kt

```diff
@@ -28,4 +31,22 @@ class ShoppingListRepository @Inject constructor(
         val entities = list.toDataLayerEntity(projectId)
         shoppingListDAO.createShoppingList(entities.first, entities.second)
     }
+
+    fun getShoppingListsForProject(projectId: Long) : Flow<List<ShoppingList>> {
+        val shoppingLists = shoppingListDAO.getShoppingListsByProjectID(projectId)
```

Comment:
Why do not request only the needed entities for each shopping list?

Reply:
I am not actually sure how the corresponding screen will look like and which information it will need. I added a function to get only the stubs without items, but depending on what we actually want to display we will have to change this again (e.g. if we want to display how many items are in each shopping list).